server/monitor/pom.xml

```diff
@@ -266,9 +273,62 @@
           <configuration>
             <excludes>
               <exclude>src/main/resources/org/apache/accumulo/monitor/resources/external/**/*</exclude>
+              <exclude>src/main/resources/org/apache/accumulo/newmonitor/dist/**/*</exclude>
+              <exclude>src/main/resources/org/apache/accumulo/newmonitor/node_modules/**/*</exclude>
+              <exclude>src/main/resources/org/apache/accumulo/newmonitor/**/*.tsbuildinfo</exclude>
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>com.github.eirslett</groupId>
+          <artifactId>frontend-maven-plugin</artifactId>
+          <version>1.15.1</version>
```

Comment:
Yea thats a good point. For how things are set up here, we require npm in order to package the React Typescript code into a single plain javascript and single css file. Those are the files that are distributed by the sever. Right now I have the `/dist` dir (which is where the packaged files are placed) in the .gitignore so things need to be built each time in order for the monitor to work. I think we could just check in the packaged files so that we dont have to include that part in our build. That way we can do what you suggested and only rebuild if changes are made to the front-end code.